distributed/cli/tests/test_tls_cli.py

```diff
@@ -38,6 +38,24 @@ def test_basic(loop):
                 wait_for_cores(c)
 
 
+def test_sni(loop):
+    with popen(["dask-scheduler", "--no-dashboard"] + tls_args) as s:
+        with popen(
+            [
+                "dask-worker",
+                "--no-dashboard",
+                "--scheduler-sni",
+                "localhost",
+                "tls://127.0.0.1:8786",
```

Comment:
Since this PR was made we've now changed how we configure ports in tests. We have an "open_port" function.

I can fix this after this is merged if you'd prefer

Reply:
> I can fix this after this is merged if you'd prefer

Thanks @graingert that would be great.

Are you happy for this to be merged?